ecs/README.md

```diff
@@ -11,20 +26,20 @@
      }
    ]
    ```
-2. Register your task definition:
+3. In the AWS CLI, type the following command to register your task definition:
```

Comment:
Instead of "In the AWS CLI", I think "At your command prompt of choice" or something to that effect.  They aren't actually in the AWS CLI when typing this.

Reply:
Or mirroring the verbiage you have below:

"Paste the command into the Linux command line. "

Reply:
OK, I thought we had talked about needing the AWS CLI for this particular step. Is it required for any of these procedures? If not, I'll also remove the paragraph that states customers should install the CLI.

Reply:
While it is true that the AWS CLI is required for the command indicated in the next step, they aren't typing the "aws ecs" command IN the AWS CLI.  They are typing this command at a Linux command prompt, which invokes the AWS CLI.